pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
```

Comment:
This is only required because tox is (currently) configured to install the project; if you add `skip_install = True` to the `[testenv]` section of the Tox configuration, pyproject.toml won't be required.